bin/test-workflow/start

```diff
@@ -19,6 +19,12 @@ Usage: ./start [options]:
                                 - Defaults to 'gke'
                                 - Supports 'jenkins'
                               All other selections are rejected
+    -a, --apps <app>          Specify which test apps to install
```

Comment:
If I'm understanding the arg parsing below, someone wanting to test multiple application types would call this like so:
```
start -a summon-sidecar secretless-broker
```
Is this accurate? Would it be clearer to have them pass in a comma-separated list, e.g.:
```
start -s "summon-sidecar,secretless-broker"
```
WDYT?

Reply:
Yeah, your flag usage is accurate here. I do like that second usage - I updated the `apps` flag to accept one comma-delimited string, and also changed the delimiter for the `INSTALL_APPS` envvar to a comma, for consistency.